packages/nuxt/src/app/composables/id.ts

```diff
@@ -26,6 +26,9 @@ export function useId (key?: string): string {
   const instanceIndex = key + ':' + instance._nuxtIdIndex[key]++
 
   if (import.meta.server) {
+    if (!import.meta.dev && instance.vnode.type && typeof instance.vnode.type === 'object' && 'inheritAttrs' in instance.vnode.type && instance.vnode.type.inheritAttrs === false) {
```

Comment:
we don't warn in dev ?

Reply:
I think we should consider warning only in dev, no? 🤔

Reply:
yes! 🤦

Reply:
```suggestion
    if (import.meta.dev && instance.vnode.type && typeof instance.vnode.type === 'object' && 'inheritAttrs' in instance.vnode.type && instance.vnode.type.inheritAttrs === false) {
```